modules/login/php/passwordreset.class.inc

```diff
@@ -73,45 +74,75 @@ class PasswordReset extends \NDB_Form
      *
      * @param array $values The values that were submitted to the page.
      *
-     * @return none
+     * @return void
      */
     function _process($values)
     {
+        $DB         =& \Database::singleton();
+        $resetLimit =10;
+
         // Always claim success to phishers.
         $this->tpl_data['success'] = 'Password reset. '
-        . 'You should receive an email within a few minutes.';
+            . 'You should receive an email within a few minutes.';
```

Comment:
@driusan @ridz1208 I think everything I said in the other thread is actually wrong given this line.

I thought that a message like this would be displayed via Javascript (aka client-side) which would make timing attacks less of a concern since you're not depending on the server finishing up work to update the front-end. Given this approach, the two of you are right -- timing attacks are a concern.